### DISCUSSION TITLE
Searchable borg index

### DISCUSSION THREAD

**Discussion by BlackLotus** [borgbackup/borg#8200] [Ideas]
Ok this will be really slow and inefficient, but it's a feature I want/need.
It's a locate for borg backups. I would like to be able to index multiple borg backups and the syntax should/could be like `locate` in the end it could be something like this
`borg locate foobar`
```
/storage/Backups/graphics::2023-02-01/graphics/foobar-1.xcf
/storage/Backups/graphics::2023-02-01/graphics/foobar-2.xcf
```
You would have to manually add repos to the index and would have to provide keys, but in theory you could encrypt the index as well...

**Comment by ThomasWaldmann** [comment]
Check the related ticket about this on the issue tracker.

Additionally, be aware that borg is not managing above the repository level. There are no commands covering multiple / all repos.

**Reply by BlackLotus** [reply]
> Check the related ticket about this on the issue tracker.

Which ticket are your referencing? I search for "search" https://github.com/borgbackup/borg/issues?q=is%3Aissue+is%3Aopen+search and index and went through all tickets, but couldn't find a ticket... Maybe I'm just blind

> Additionally, be aware that borg is not managing above the repository level. There are no commands covering multiple / all repos.

Yeah I know that of course, I just hoped for a functionality to make it possible/easier. Right now I'm considering a simple solution for me that stores repos and passwords in a file and creates compressed "index" files that can be searched with fzf and grep.... which is less than ideal.

**Reply by ThomasWaldmann** [reply]
I guess the ticket was closed, so you need to also search in the closed tickets.

**Reply by krooijers** [reply]
Possibly: https://github.com/borgbackup/borg/issues/6074 ?

**Reply by ThomasWaldmann** [reply]
yeah, there i gave feedback with my impressions from playing with whoosh search lib.
